diff --git legacy_fortran/triaxnnls_CRcut.f90 legacy_fortran/triaxnnls_CRcut.f90
index e2a7091..13e1415 100755
--- legacy_fortran/triaxnnls_CRcut.f90
+++ legacy_fortran/triaxnnls_CRcut.f90
@@ -1090,7 +1090,7 @@ end subroutine donnls_galahad
    !CALL PRESOLVE_apply( p, pcontrol, inform, pdata )
    IF ( inform%status /= 0 ) STOP
 
-   CALL QPB_initialize( data, control)!,info )         ! Initialize control parameters                                          
+   CALL QPB_initialize( data, control,info )         ! Initialize control parameters                                          
    !   control%rho_g = 1.0_dp ; control%rho_b = 1.0_dp        ! initial penalty parameters
    control%infinity = infinity                  ! Set infinity
    control%print_level = 1
